### DISCUSSION TITLE
Problems about converting the waymo dataset

### DISCUSSION THREAD

**Discussion by JiSuanJiDaWang** [open-mmlab/mmdetection3d#2049] [Q&A]
I found that the `_calculate_num_points_in_gt
` is extremely slow in my pc, is that reasonable? Seems it got stuck in this function.
![image](https://user-images.githubusercontent.com/82489384/203455748-0d4cf25c-551e-42bf-9eaa-9bd49ef4d717.png)

**Comment by JingweiZhang12** [comment]
In this case, suggest not using multi-process.

**Reply by JiSuanJiDaWang** [reply]
Hi, I found that the problem was caused by the `   ret_infos = mmengine.track_parallel_progress(self.calculate_single,
                                                     infos, self.num_worker)` as you said. I set the number of workers to 1, but it still stuck there. Is there any way to just run the `calculate_single`. I used the following code to replace the original one, it works.
![image](https://user-images.githubusercontent.com/82489384/203485610-0aa43255-2062-4dd0-9c1b-fdcfbcd4f8b3.png)
